scripts/wdl/pon_gatk_workflow.wdl

```diff
@@ -0,0 +1,358 @@
+                --reference ${ref_fasta} --input ${input_bam} --binsize ${wgsBinSize}; \
+          else ln -s ${coverage_file} ${entity_id}.coverage.tsv; ln -s ${target_file} ${entity_id}.coverage.tsv.targets.tsv; \
+        fi
+    }
+
+    output {
+        File gatk_coverage_file = "${entity_id}.coverage.tsv"
+        File gatk_target_file = "${entity_id}.coverage.tsv.targets.tsv"
+    }
+}
+
+# Helper task to aggregate array of targets, that scatter block outputs, into a single file
+task AggregateTargetFiles {
+    Array[File]+ target_file_list
+
+    command <<<
+        ln -s ${target_file_list[0]} targets.tsv
+    >>>
+
+    output {
+        File target_file = "targets.tsv"
+    }
+}
+
+# Combines a set of read counts input files into a single multi read count column output file
+task CombineReadCounts {
+    String combined_entity_id
+    Array[File] coverage_file_list
+    Int max_open_files
+    File gatk_jar
+    Int mem
+
+    command {
+        java -Xmx${mem}g -jar ${gatk_jar} CombineReadCounts --input ${sep=" --input " coverage_file_list} \
+         --maxOpenFiles ${max_open_files} --output ${combined_entity_id}.tsv
+    }
+
+    output {
+        File combined_read_counts = "${combined_entity_id}.tsv"
+    }
+}
+
+# Add new columns to an existing target table with various targets
+# Note that this task is optional
+task AnnotateTargets {
+    String entity_id
+    File target_file
+    File gatk_jar
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    Boolean enable_gc_correction
+    Int mem
+
+    # If GC correction is disabled then, an empty file gets passed downstream
+    command {
+        if [ ${enable_gc_correction} = true ]; \
+          then java -Xmx${mem}g -jar ${gatk_jar} AnnotateTargets --targets ${target_file} --reference ${ref_fasta} --output ${entity_id}.annotated.tsv; \
+          else touch ${entity_id}.annotated.tsv; \
+        fi
+    }
+
+    output {
+        File annotated_targets = "${entity_id}.annotated.tsv"
+    }
+}
+
+# Correct coverage for sample-specific GC bias effects
+# Note that this task is optional
+task CorrectGCBias {
+    String entity_id
+    File coverage_file
+    File annotated_targets
+    File gatk_jar
+    Boolean enable_gc_correction
+    Int mem
+
+    # If GC correction is disabled then, coverage file gets passed downstream unchanged
+    command {
+        if [ ${enable_gc_correction} = true ]; \
+          then java -Xmx${mem}g -jar ${gatk_jar} CorrectGCBias --input ${coverage_file} \
+           --output ${entity_id}.gc_corrected_coverage.tsv --targets ${annotated_targets}; \
+          else ln -s ${coverage_file} ${entity_id}.gc_corrected_coverage.tsv; \
+        fi
+    }
+
+    output {
+        File coverage_file_gcbias_corrected = "${entity_id}.gc_corrected_coverage.tsv"
+    }
+}
+
+# Create panel of normals given a collection of read-counts for control samples
```

Comment:
Just a note, typically I hyphenate "read count" only when using it as a compound modifier, e.g. "read-count collection"